030-ConversationalAI/Coach/Readme.md

```diff
@@ -0,0 +1,29 @@
+
+# What The Hack - Conversational AI Attach Play
+
+## Introduction
+Welcome to the coach's guide for Conversational AI Attach Play What The Hack. For each challenges we have a solution tips and notes for you coach. You can get an end Bot Composer solution in this folder under Bot Composer Project. You can open that Bot Composer Project on your local Bot Framework Composer application to check the sample core logic this Bot has.
```

Comment:
slight tweak: "For each challenge, we have solutions, tips and notes for you as the coach.

Reply:
Top level one is not needed. let me update it